docs/001_develop/03_client-capabilities/009_presentation/skeleton.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: 'Skeleton'
+sidebar_label: 'Skeleton'
+id: client-presentation-skeleton
+keywords: [presentation, skeleton]
+tags:
+- presentation
+- skeleton
+sidebar_position: 9
+---
+
+import SkeletonDemo from './examples/skeleton.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Skeleton
+
+The `skeleton` component is used as a visual placeholder for an element while it is in a loading state.
+It usually presents itself as a simplified wireframe-like version of the UI it is representing.
+
+Use cases:
+- Indicating loading state
+
+## Example
+
+<SkeletonDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+  <TabItem value="genesis">
+
+    Declaration
+    ```html
+    <rapid-skeleton></rapid-skeleton>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+      name: 'my-element',
+      template: html`
+        <rapid-skeleton></rapid-skeleton>
+      `,
+    })
+    export class MyElement extends GenesisElement {
+    }
+    ```
+
+  </TabItem>
+  <TabItem value="react">
+
+  Declaration
+  ```html
+    <rapid-skeleton></rapid-skeleton>
+  ```
+  Usage
+  ```typescript
+  export function MyComponent() {
+    return (
+      <rapid-skeleton></rapid-skeleton>
+    );
+  }
+  ```
+</TabItem>
+  <TabItem value="angular">
+
+  Declaration
+  ```html
+    <rapid-skeleton></rapid-skeleton>
+  ```
+
+  Usage
+  ```typescript
+  import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+  @Component({
+    selector: 'my-root',
+    template: `
+        <rapid-skeleton></rapid-skeleton>
+        `,
+    standalone: true,
+    schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  })
+  export class AppComponent {
+  }
+  ```
+</TabItem>
+</Tabs>
+
+## API
+
+Property and attribute binding examples for Genesis Component syntax. Closing tag omitted.
+
+### Attributes
+
+<table>
+  <thead><tr><th>Name</th><th>Type</th><th>Description</th><th>Example</th></tr></thead>
+  <tbody>
+    <tr>
+      <td>fill</td>
+      <td><code>string</code></td>
```

Comment:
Is there a union of string values that we know that can go here? `string` itself isn't very clear.

Reply:
Not sure on this one, I assumed it was css fill 
https://developer.mozilla.org/en-US/docs/Web/CSS/fill
its not too clear from the code to me though.